app/lib/dttp/batch_request.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Dttp
+  class BatchRequest
+    class Error < StandardError; end
```

Comment:
Does this needs to sit on its own file?

Reply:
You mean `class Error < StandardError; end`?

Reply:
I don't really mind leaving these inside the classes that throw them but we did move `DttpIdNotReturnedError` out into its own file after @defong pulled me up about it on another PR 🤷

Reply:
I think we should keep it here - it feels like a natural home for it. Besides, it's just a one-liner.

I would even go further and suggest getting rid of `app/models/dttp_id_not_returned_error.rb` (doesn't belong in this directory IMO) and just put `class DttpIdNotReturnedError < StandardError; end` inside `Dttp::OdataParser` where it's used.